### PR TITLE
Improvements to the performance testing harness

### PR DESCRIPTION
**Pull request by kciesielski**
1. Allow running tests for server groups like `pekko.*`, `vertx.future.*`, etc. 
2. Remove `*` for running all servers - there's too many of them anyway
3. Add servers with ServerLog interceptor enabled

### CODE REVIEW

File: perf-tests/src/main/scala/sttp/tapir/perf/vertx/Vertx.scala [hunk 1]
```diff
@@ -96,6 +96,9 @@ object VertxRunner {
 }
 
 object TapirServer extends ServerRunner { override def start = VertxRunner.runServer(Tapir.route(1)) }
-object TapirMultiServer extends ServerRunner { override def start = VertxRunner.runServer(Tapir.route(127)) }
+object TapirMultiServer extends ServerRunner { override def start = VertxRunner.runServer(Tapir.route(128)) }
+object TapirInterceptorMultiServer extends ServerRunner {
+  override def start = VertxRunner.runServer(Tapir.route(128, withServerLog = true))
+}
 object VanillaServer extends ServerRunner { override def start = VertxRunner.runServer(Vanilla.route(1)) }
-object VanillaMultiServer extends ServerRunner { override def start = VertxRunner.runServer(Vanilla.route(127)) }
```
Comment (kciesielski) [inline perf-tests/src/main/scala/sttp/tapir/perf/vertx/Vertx.scala:101]: **issue:** This is strange, `SimpleGetMultiRoute` tests should have been failing with 127 endpoints. I need to re-run them and double check the results now.

Comment (adamw) [reply perf-tests/src/main/scala/sttp/tapir/perf/vertx/Vertx.scala:101]: because you've been hitting endpoint 127 and they had numbers 0..126? :)

**Review by adamw — APPROVED** [review]

File: perf-tests/src/main/scala/sttp/tapir/perf/vertx/Vertx.scala (hunk 1 above)
Comment (kciesielski) [reply perf-tests/src/main/scala/sttp/tapir/perf/vertx/Vertx.scala:101]: Yeah :) But Gatling still logs throughput, even if it gets 404s ;)

Comment (adamw) [reply perf-tests/src/main/scala/sttp/tapir/perf/vertx/Vertx.scala:101]: maybe we can expect 200 in the scenario?

**Pull request merged**